File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/SimpleDependency_ViewModelTestEnvironmentImpl.cpp
#include "SimpleDependency_ViewModelTestEnvironmentImpl.h"
#include <memory>
#include "SimpleDependency_ViewModel.h"
#include "MessageBus.h"
#include "LoggerSpy.h"

namespace dependencies
{
  void SimpleDependency_ViewModelTestEnvironmentImpl::Init()
  {
  }
  std::shared_ptr<SimpleDependency_ViewModel> SimpleDependency_ViewModelTestEnvironmentImpl::BuildSut()
  {
    return nullptr;
  }
  std::shared_ptr<MessageBus> SimpleDependency_ViewModelTestEnvironmentImpl::getMessageBus()
  {
    return nullptr;
  }
  std::shared_ptr<LoggerSpy> SimpleDependency_ViewModelTestEnvironmentImpl::getLogger()
  {
    return nullptr;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/SimpleDependency_ViewModelTestEnvironmentImpl.cs
/// <filename>
///     SimpleDependency_ViewModelTestEnvironmentImpl.cs
/// </filename>
using dependencies;

namespace dependencies
{
    public class SimpleDependency_ViewModelTestEnvironmentImpl : dependencies.SimpleDependency_ViewModelTestEnvironment
    {
        public override void Init()
        {
        }

        public override dependencies.SimpleDependency_ViewModel BuildSut()
        {
            return null;
        }

        public override dependencies.MessageBus getMessageBus()
        {
            return null;
        }

        public override dependencies.LoggerSpy getLogger()
        {
            return null;
        }
    }
}
